yarn.lock

```diff
@@ -1255,7 +1255,7 @@
   resolved "https://registry.yarnpkg.com/@hapi/hoek/-/hoek-9.2.1.tgz#9551142a1980503752536b5050fd99f4a7f13b17"
   integrity sha512-gfta+H8aziZsm8pZa0vj04KO6biEiisppNgA1kbJvFrrWu9Vm7eaUEy76DIxsuTaWvti5fkJVhllWc6ZTE+Mdw==
```

Comment:
We strictly reference `@hapi/hoek@9.2.0` but `9.2.1` was pulled in because `@hapi` packages reference `@hapi/hoek@9.x.x`.

I guess this kind of duplication will happen every time some strictly referenced package is updated.